CONTRIBUTING.md

```diff
@@ -20,26 +20,90 @@
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+### Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at support@digi.me. All
+complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident.
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
+## Open Development
 All work on our SDK happens directly on Github. Core team members and external contributors send pull requests which go through the same review process. 
 
-### Semantic Versioning
+## Semantic Versioning
 Our SDK follows semantic versioning. Patch versions are reserved for bug fixes, minor versions for new features and major version for any breaking changes.
 
-### Branch Organisation
+## Branch Organisation
 Please submit all changes directly to the master branch. We do our best to keep master in good shape, with all tests passing. Master could potentially have breaking changes to your application if we're working on the next major release.
 
-### Workflow and Pull Requests
-For non-trivial changes, please open an issue with a proposal for a new feature or refactoring before starting on the work. We don't want you to waste your efforts on a pull request that we won't want to accept. The core team will be monitoring for all pull requests. We require that tests to be included with your pull requests if you've added code that should be tested. Two approvers are needed before code can be merged.
+## Workflow and Pull Requests
+For non-trivial changes, please open an issue with a proposal for a new feature or refactoring before starting on the work. We don't want you to waste your efforts on a pull request that we won't want to accept. The core team will be monitoring for all pull requests. We require that tests to be included with your pull requests if you've added code that should be tested. Two approvers are needed before code can be merged. We also require that linting to be performed on your branch before submitting a pull request.
 
-### Development
-To start working on the SDK, fork, then clone the repo:
+## Development
+To start working on the SDK, fork, then clone their repo:
 
 `git clone https://github.com/digime/digime-sdk-js.git`
```

Comment:
Sorry, I wasn't clear in my previous comment. The github url should point to their forked repo (not ours). As such; 

To start working on the SDK, fork, then clone the repo:
`git clone https://www.github.com/<your_username>/digime-sdk-js.git`